### ISSUE TITLE
Store command encrypts user's local copy

### ISSUE THREAD

**Issue by finoradin** [filecoin-project/starling#29]
**Steps to reproduce:**
1. create `foo.txt` in `~/`
2. `starling store foo.txt`
3. `ls ~/` and **sometimes** `foo.txt` will be gone and all you will see remaining is `foo.txt-encrypted`

**Expected behavior**
- DO NOT delete the original file
- The encrypted copy should IMHO not be stored along side the source file (perhaps stored somewhere in Starling app's files hidden from view? or is there some system dir for this sort of thing?) and should be temporary – i.e. once file has sealed it should be deleted from local storage

**Comment by alexandrumatei36** [comment]
I can say for sure that the original file is not deleted

**Comment by alexandrumatei36** [comment]
> The encrypted copy should IMHO not be stored along side the source file

I agree. Encrypted file is kept because it can't be retrieved later from Filecoin if it's deleted from disk. This is a known bug and we're waiting for a fix
